src/api/Controllers/BoreholeController.cs

```diff
@@ -33,19 +36,39 @@ public async override Task<ActionResult<Borehole>> CreateAsync(Borehole entity)
             PublishedTabs = new(),
         };
 
-        // temporarily add legacy workflow until WorkflowV2 is fully implemented
+        if (!await boreholePermissionService.HasUserRoleOnWorkgroupAsync(HttpContext.GetUserSubjectId(), entity.WorkgroupId, Role.Editor).ConfigureAwait(false))
```

Comment:
PP: `HttpContext.GetUserSubjectId()` in Variable auslagern und weiter unten wiederverwenden.